onnxscript/rewriter/min_max_to_clip.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Does the following transformation:
+- Min(Min(X)) -> Min(X)
+- Max(Max(X)) -> Max(X)
+- Min(Max(X)) -> Clip(X)
+- Max(Min(X)) -> Clip(X)
```

Comment:
Could you expand the docstring to describe the scalar and number of inputs requirements? I would also add them to the class docstrings.